src/modules/pos/validator/components/ChangeCommission/Dialog/ChangeCommissionDialog.js

```diff
@@ -0,0 +1,33 @@
+/* istanbul ignore file */
+import React from 'react';
+import { withRouter } from 'react-router';
+import { useTranslation } from 'react-i18next';
+
+import MultiStep from '@common/components/OldMultiStep';
+import TxSignatureCollector from '@transaction/components/TxSignatureCollector';
+import Dialog from '@theme/dialog/dialog';
+import { ChangeCommissionForm as Form } from '../Form';
+import Summary from '../Summary';
+import Status from '../Status';
+import styles from './ChangeCommissionDialog.css';
+
+export const ChangeCommission = ({ history }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Dialog hasClose size="sm">
+      <MultiStep
+        className={styles.multiStep}
+        prevPage={history.goBack}
+        backButtonLabel={t('Back')}
+      >
+        <Form />
+        <Summary />
+        <TxSignatureCollector />
+        <Status />
```

Comment:
Do we have any issue or ticket for also displaying this error to the user:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/8784876/218072507-5c1b73ae-459c-4f7c-921c-1b04623443ba.png">

I could only test to change the commission one time

Reply:
yeah its correct you can not change commission in short period